homeworks/sargis_vardanyan/25-11-2021/loops/LoopsTask6.java

```diff
@@ -0,0 +1,37 @@
+package loops;
+
+import java.util.Scanner;
+
+/*
+  x and y are natural numbers. Assign 5 to a variable z, if
+  (x + y) is a simple number, otherwise assign 6 and print. 
+*/
+public class LoopsTask6{
+
+public static void main(String [] args){
+
+    System.out.println("Give value to x :");
+    Scanner scanner = new Scanner(System.in);
+    int x = scanner.nextInt();
+    System.out.println("Give value to y :");
+    int y = scanner.nextInt();
+    int z = 5;
+    
+    if((x + y) % 2 == 0 && (x + y) != 2){
+        z = 6;  
+    } else {
```

Comment:
I see that you are trying to do some optimization, but this doesn't worth it, as the first item in for should be 2.